cost/aws/unused_volumes/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # Changelog
 
+## v7.4
```

Comment:
For changes or corrections to the README, there is no need to update the CHANGELOG or change the version number. This is only for changes to the actual policy.